test/fixtures/apilinks/reverse.json

```diff
@@ -0,0 +1,3 @@
+{
+  "asserts.ok": "reverse.js#L10"
```

Comment:
Any chance we could figure out that this should link to L5? (It seems too tricky to me but I figured I’d ask :slightly_smiling_face:)

Reply:
I'll take a look.  I'm inclined to only do that for statements of the form:

    x.y = y;

In other words, **not** for the following (which would be an alias):

    x.y = z;

Thoughts?

Reply:
I think that sounds reasonable, yeah :)